app/controllers/concerns/response.rb

```diff
@@ -0,0 +1,7 @@
+module Response
+  extend ActiveSupport::Concern
+
+  def json_response(object, status = :ok)
+    render json: object, status:
+  end
+end
```

Comment:
- [ ] Kindly, as you are using Concerns to make your code reusable and reduce its complexity(make it less complicated) make sure that you implement very well the encapsulation that extends the `ActiveSupport::Concern` . For now when I run your project I have this error below:
![image](https://user-images.githubusercontent.com/71874425/184131707-3398d8fe-7fa9-466e-b60b-b691228cd43e.png)
Kindly, fix this issue.